### PR TITLE
fix focus effect on up-tree

### PR DESCRIPTION
**Pull request by DennyDai**
Hi, I found another problem like the problem before.
the same problem still exists in the up-tree row.

### CODE REVIEW

File: src/inject.js [hunk 1]
```diff
@@ -235,6 +236,7 @@ var domUtils = {
             ns[i].insertAdjacentHTML('afterend', '<td></td>');
           }
         }
+        uptree[3].insertAdjacentHTML('afterend', '<td></td>');
```
Comment (softvar) [inline src/inject.js]: This will throw error if uptree is not defined, which is a valid case

File: src/inject.js [hunk 2]
```diff
@@ -207,6 +207,7 @@ var domUtils = {
   addFileSizeAndDownloadLink: function  () {
     var links = document.querySelectorAll('tr.js-navigation-item > td.content a');
     var ns = document.querySelectorAll('tr.js-navigation-item > td.age');
+    var uptree = document.querySelectorAll('tr.up-tree > td');
 
     if (ns.length && ns.length === links.length) { // verify length for showing in-sync
       apiUtils.getRepoContent(function (data) {
```
Comment (softvar) [inline src/inject.js]: Add this

```

if (uptree && uptree[3]) {
  uptree[3].insertAdjacentHTML('afterend', '<td class=""></td>');
}
```

just after the if line statement

**Comment by DennyDai** [comment]
Changed already, is that work?

File: src/inject.js [hunk 3]
```diff
@@ -237,6 +238,10 @@ var domUtils = {
         }
       });
     }
+    if (uptree && uptree[3]) {
```
Comment (softvar) [inline src/inject.js]: Now the issue is:

This code will remove all the elements having `download` class, right?
But, as per the above code, which makes a call and then do something, which will always be called after this piece od code.
So, what will happen is: that code will internally executes `utils.removePrevInstancesOf('.download');` which in turn will remove the newly added `td` for uptree.

So, this code breaks up.

Solution is:

Add the below code:

```
if (uptree && uptree[3]) {
   uptree[3].insertAdjacentHTML('afterend', '<td class="download"></td>');
}
```

after these lines:

```
...
data = utils.sortFileStructureAsOnSite(data);

        if (!data) { return; }

        utils.removePrevInstancesOf('.download'); // remove before adding new ones
```

which will handle it.

Also, please check once before pushing. The procedure is simple and is documented in the README.md.
1. create a branch and then work, don't commit to `master` directly
2. run `npm install` once and then gulp watch
3. do your changes, update the extension by clicking on refresh button, after loading it as unpacked - `chrome://extensions`
4. Verify the changes
5. Push them, happy coding!

**Comment by softvar** [comment]
Thanks for the patch!

**Pull request merged**